infra/src/main/kotlin/fi/fta/geoviite/infra/inframodel/InfraModelService.kt

```diff
@@ -186,9 +186,9 @@ class InfraModelService @Autowired constructor(
             project = planProject,
             author = planAuthor,
             application = application,
-            planPhase = extraInfoParameters?.planPhase ?: plan.planPhase,
-            decisionPhase = extraInfoParameters?.decisionPhase ?: plan.decisionPhase,
-            measurementMethod = extraInfoParameters?.measurementMethod ?: plan.measurementMethod,
+            planPhase = extraInfoParameters?.planPhase,
+            decisionPhase = extraInfoParameters?.decisionPhase,
+            measurementMethod = extraInfoParameters?.measurementMethod,
             message = extraInfoParameters?.message ?: plan.message,
             planTime = overrideParameters?.createdDate ?: plan.planTime,
```

Comment:
Näihin jäljelle jääneisiin elvistelykenttiin voisi ehkä heittää vielä jonkun koodikommentin, että miksi nämä ovat näin, kun muut ovat nullable

Reply:
Lisäilin kommentin. Näissä toki haasteena, että liian geneerinen kommentti ei kerro tarpeeksi ja liian täsmällinen ei pysy muutosten kanssa synkronissa.

Varmaan tuo ainakin auttaa huomaamaan, että "samassa paikassa" on kahta erityyppistä menettelyä. Erilaisten operaatioiden pitäminen samassa paikassa onkin sitten toinen pulma...